packages/teleport/src/Nodes/Nodes.tsx

```diff
@@ -108,14 +108,11 @@ export function Nodes(props: State) {
 }
 
 const emptyStateInfo: EmptyStateInfo = {
-  title: 'ADD YOUR FIRST SERVER',
-  description: (
-    <Text>
-      Instant SSH access for engineers with native security and compliance.
-    </Text>
-  ),
-  buttonText: 'ADD SERVER',
-  videoLink: 'https://www.youtube.com/watch?v=tUXYtwP-Kvw',
+  title: 'Add your first Linux server to Teleport',
```

Comment:
`Add your first Linux server to Teleport` -> `Add your first SSH Server to Teleport` 

It doesn't have to be Linux afaik (you can add a MacBook as a node for example)